ui-kit/src/main/kotlin/org/p2p/uikit/components/UiKitTip.kt

```diff
@@ -19,13 +22,49 @@ private enum class TipPosition {
     BOTTOM
 }
 
+enum class TipColor(
+    @ColorRes val backgroundColor: Int,
+    @ColorRes val titleTextColor: Int,
+    @ColorRes val buttonColor: Int,
+    @ColorRes val buttonNextTextColor: Int,
+    @ColorRes val buttonSkipTextColor: Int
+) {
+    SNOW(
+        backgroundColor = R.color.bg_snow,
+        titleTextColor = R.color.text_night,
+        buttonColor = R.color.bg_lime,
+        buttonNextTextColor = R.color.text_night,
+        buttonSkipTextColor = R.color.text_night
+    ),
+    NIGHT(
+        backgroundColor = R.color.bg_night,
+        titleTextColor = R.color.text_snow,
+        buttonColor = R.color.bg_lime,
+        buttonNextTextColor = R.color.text_night,
+        buttonSkipTextColor = R.color.text_lime
+    ),
+    LIME(
+        backgroundColor = R.color.bg_lime,
+        titleTextColor = R.color.text_night,
+        buttonColor = R.color.bg_night,
+        buttonNextTextColor = R.color.text_lime,
+        buttonSkipTextColor = R.color.text_night
+    )
+}
+
+data class TipConfig(
+    val tipColor: TipColor,
+    val withArrow: Boolean = true
+)
+
 /**
  * Shows PopupWindow with arrow
  * PopupWindow appears natively above or under the view
  * Arrow is shown only on top or bottom of PopupWindow
  * Arrow is pointing on the center of view
  */
 fun View.showTip(
+    tipConfig: TipConfig,
```

Comment:
why other parameters are not for TipConfig too? :)
looks kinda strange from API view

Reply:
im okay with that